site/content/3.12/data-science/arangographml/getting-started.md

```diff
@@ -184,305 +284,500 @@ The featurization specification asks that you input the following:
+)
 
-# Tip: Sort by accuracy
-models.models.sort(key=(lambda model : model.model_statistics["test"]["accuracy"]) , reverse=True)
-# The most accurate model is the first in the list
-model = models.models[0]
-print(model)
+print(best_model)
 ```
 
-**Expected output:**
+**Example Output:**
 ```py
-{'job_id': 'f09bd4a0-d2f3-5dd6-80b1-a84602732d61',
- 'model_display_name': 'Node Classification Model',
- 'model_id': '123',
- 'model_name': 'Node Classification Model '
-               '123',
- 'model_statistics': {'_id': 'devperf/123',
-                      '_key': '123',
-                      '_rev': '_gkUc8By--_',
-                      'run_id': '123',
-                      'test': {'accuracy': 0.8891242216547955,
-                               'confusion_matrix': [[13271, 2092],
-                                                    [1276, 5684]],
-                               'f1': 0.9,
-                               'loss': 0.1,
-                               'precision': 0.9,
-                               'recall': 0.8,
-                               'roc_auc': 0.8},
-                      'timestamp': '2023-09-01T17:32:05.899493',
-                      'validation': {'accuracy': 0.9,
-                               'confusion_matrix': [[13271, 2092],
-                                                    [1276, 5684]],
-                               'f1': 0.85,
-                               'loss': 0.1,
-                               'precision': 0.86,
-                               'recall': 0.85,
-                               'roc_auc': 0.85}},
- 'target_collection': 'target_collection_name',
- 'target_field': 'label_field'}
+{
+  "job_id": "691ceb2f-1931-492a-b4eb-0536925a4697",
+  "model_id": "02297435-3394-4e7e-aaac-82e1d224f85c",
+  "model_statistics": {
+      "_id": "devperf/123",
+      "_key": "123",
+      "_rev": "_gkUc8By--_",
+      "run_id": "123",
+      "test": {
+          "accuracy": 0.8891242216547955,
+          "confusion_matrix": [[13271, 2092], [1276, 5684]],
+          "f1": 0.9,
+          "loss": 0.1,
+          "precision": 0.9,
+          "recall": 0.8,
+          "roc_auc": 0.8,
+      },
+      "validation": {
+          "accuracy": 0.9,
+          "confusion_matrix": [[13271, 2092], [1276, 5684]],
+          "f1": 0.85,
+          "loss": 0.1,
+          "precision": 0.86,
+          "recall": 0.85,
+          "roc_auc": 0.85,
+      },
+  },
+  "target_collection": "Event",
+  "target_field": "label",
+}
 ```
 
-### Prediction
+## Prediction
+
+After selecting a model, a Prediction Job can be created. The Prediction Job
+will generate predictions and persist them to the source graph in a new
+collection or within the source document.
+
+The Prediction Specification requires the following:
+- `project_name`: The top-level project to which all the experiments will link back.
+- `database_name`: The database name the source data is in.
+- `model_id`: The model ID to use for generating predictions.
```

Comment:
TODO: There are more parameters than this. Would it be better to list them here? Or just have a link pointing to the API Specification of the Prediction service?

Reply:
Are these the only required parameters, and the rest are optional?

Reply:
If that's the case, should be fine to just add a link to the APIs specs.